### PR TITLE
Upgrade workflows

### PR DESCRIPTION
**Pull request by Gauvino**
#### Description
Upgrade github action to theyr latest version and change the ubuntu version to runs-on

#### To-Dos

- [ ] Successful build `yarn build`
- [ ] Translation keys `yarn i18n:extract`
- [ ] Database migration (if required)

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -12,11 +12,11 @@ jobs:
   test:
     name: Lint & Test Build
     if: github.event_name == 'pull_request'
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
     container: node:16.17-alpine
```
Comment (Gauvino) [inline .github/workflows/ci.yml:16]: @Fallenbagel does this need to be changed to a newer version or not I was not sure?

Comment (Gauvino) [reply .github/workflows/ci.yml:16]: node:17-alpine maybe this one ?

**Comment by bdovaz** [comment]
@Gauvino as for the actions, the ultimate solution is to configure dependabot, no? 

https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#github-actions

**Comment by Gauvino** [comment]
> @Gauvino as for the actions, the ultimate solution is to configure dependabot, no?
> 
> https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#github-actions

Yes of course it would the best solution.

**Comment by Fallenbagel** [comment]
Supercedwd by: 59dabed380cde4fc5e3936b8ba05f018580f72da

**Pull request closed without merging**